### PR TITLE
Add icon for Wechat

### PR DESCRIPTION
**Pull request by Uni-Rider**
## Description

https://play.google.com/store/apps/details?id=com.tencent.mm

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* App Name (`package.name`) - added from icon request form
* App Name (`package.name`)
* App Name (`package.name`)

### Icons updated
* App Name (`package.name`)
* App Name (`package.name`)

### Icons linked
* App Name (linked `package.name` to `@drawable/package`)
* App Name (linked `package.name` to `@drawable/package`)

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -1147,4 +1147,5 @@
     <icon drawable="@drawable/zepp_life" package="com.xiaomi.hm.health" name="Zepp Life" />
     <icon drawable="@drawable/zepto" package="com.zeptoconsumerapp" name="Zepto" />
     <icon drawable="@drawable/zoom" package="us.zoom.videomeetings" name="Zoom" />
+    <icon drawable="@drawable/wechat" package="com.tencent.mm" name="wechat" />
```
Comment (Goooler) [inline app/src/main/res/xml/grayscale_icon_map.xml:1150]:
```suggestion
    <icon drawable="@drawable/wechat" package="com.tencent.mm" name="WeChat" />
```

**Review by Chefski — CHANGES_REQUESTED** [review]
Hey! The icon goes outside the 160x160 px content area size. Could you decrease the icon size so it fits right into it?

**Comment by Uni-Rider** [comment]
> Hey! The icon goes outside the 160x160 px content area size. Could you decrease the icon size so it fits right into it?

The icon size has been decreased to 160 px

**Review by Chefski — CHANGES_REQUESTED** [review]
The outline now has a stroke of just over 11px instead of 12pox. Mind fixing it?

**Comment by Uni-Rider** [comment]
> The outline now has a stroke of just over 11px instead of 12pox. Mind fixing it?

Can you please tell me how to adjust the stroke exactly to 12 px? I am a novice

**Comment by Chefski** [comment]
Not sure how to tell you to be honest because you have and outside and inside line so you're currently using 7px. I'm not familiar with Inkscape either. But if you look at other icon they only have one path/line

**Comment by Uni-Rider** [comment]
> Not sure how to tell you to be honest because you have and outside and inside line so you're currently using 7px. I'm not familiar with Inkscape either. But if you look at other icon they only have one path/line

I have changed the icon to single path, please check it. but the eyes seen from here are holes, if opened by IE, they are black dots, don't know why

**Review by Chefski — APPROVED** [review]
Thanks

**Pull request merged**